pages/participate.html

```diff
@@ -0,0 +1,72 @@
+            </p>
+            <p>
+                The survey has been designed around a few ground rules:
+                <ul>
+                    <li>No personal data and no data to track a specific user.</li>
+                    <li>The user is in control: you can opt-out at any time, and review the survey result at any time (via "Game Options" in-game).</li>
+                    <li>Minimize how often we send survey results.</li>
+                </ul>
+            </p>
+            <p>
+                With this in mind, we collect the following information of every game:
+                <ul>
+                    <li>The settings as they are at the end of the game.</li>
+                    <li>The NewGRFs used by the game.</li>
+                    <li>The companies, whether they are human or AI, and in case of AI: what script was running. Additionally, how many vehicles and infrastructure each company has.</li>
+                    <li>How long the game has been running.</li>
+                    <li>Basic information about your OS / CPU.</li>
+                </ul>
+            </p>
+            <p>
+                This information is used on this website to show statistics about how OpenTTD is used.
+            </p>
+        </div>
+    </div>
+    <div class="section-header">
+        <h3>Privacy statement</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>
+                As mentioned in the last section and introduction, the survey is designed to be privacy friendly.
+                This means the following:
+                <ul>
+                    <li>
+                        We do not track users.
+                        With this we mean that no information is collected that can be used to track a specific user.
+                        Everything is as generic as possible.
+                    </li>
+                    <li>
+                        Results are only published when sufficient surveys (N > 100) have been collected.
+                        This is to prevent that a single user can be identified on small samples.
+                    </li>
+                    <li>
+                        We do not store any information other than the survey results.
+                        This means we do not store IP addresses, or any other information that can be used to identify a user, next to the survey results.
+                    </li>
+                </ul>
+            </p>
+            <p>
+                To prevent abuse of this system, there are several measures in place.
+                For example, you cannot send a survey result more than once every minute.
+                This is done based on the IP address, but that is handled by a system that is not connected directly to the handling of survey results.
```

Comment:
```suggestion
                This is done based on the IP address, but by a separate system that handles the results themselves. Your IP address is never stored with the survey results.
```

Reply:
> This is done based on the IP address, but by a separate system that handles the results themselves. Your IP address is never stored with the survey results.

The above line would give a contradiction; the corrected version is (changes in bold):
> This is done based on the IP address, but by a separate system that **does not** handle the results themselves. Your IP address is never stored with the survey results.